Created Tests for ViewScores Operations

Funkey08 opened the pull request:
I've created some tests involving ViewScores, such as properly updating the UserScores csv file using the ViewScores DAO and ensuring that the ViewScoresPresenter and ViewScoresState correctly return the proper success message. Using these tests, I actually discovered that there were some minor errors with ViewScoresPresenter involving how the success message was formatted, so I've also updated that class accordingly.

dysfunctioned reviewed (approved):
Everything look quite good. I think that these tests cover the use case very well.

Funkey08 commented:
Cleaned up relevant files and updated tests for ViewScores for 100% use case coverage:

- Tests for ViewScores have been created for use-cases and interface_adapters
- ViewScores tests are now all in the correct file directories
- ViewManagerModel has been refactored to ViewModelManager; all relevant files have been updated.
- ReturnHome's use case should now be properly testable using ReturnHomeUseCaseTest; this needs to be merged to the main to make sure though

manimeh commented:
Everything looks amazing, good job! Your testing is very thorough, it covers every aspect of your use case.
